templates/docs/examples/patterns/strips/shallow.html

```diff
@@ -3,16 +3,14 @@
 
 {% block standalone_css %}patterns_strip{% endblock %}
 
+{% set is_paper = True %}
 {% block content %}
-<section class="p-strip--light is-shallow">
-  <div class="row u-vertically-center">
-    <div class="col-8">
+<section class="p-strip is-shallow">
```

Comment:
Should highlighted strip be used here? I see that the previous examples that use `p-strip--light` are being changed to `p-strip--highlighted`

Reply:
It doesn't matter here. It's an example of shallow strip, so `is-shallow` is what matters (and not using deprecated `--light` one).

But good catch! Thanks for commenting on this.